opentelemetry/Cargo.toml

```diff
@@ -34,6 +34,7 @@ serde = { version = "1.0", features = ["derive", "rc"], optional = true }
 thiserror = "1"
 tokio = { version = "1.0", default-features = false, features = ["rt", "time"], optional = true }
 tokio-stream = { version = "0.1", optional = true }
+crossbeam-channel = { version = "0.5", optional = true }
```

Comment:
Do we use crossbeam because the `crossbeam_channel::Sender` is `Sync`?

Reply:
More for performance, could impl with std channel as well and have crossbeam be an opt-in feature for people worried about the additional dependency maybe?

Reply:
I think it's fine. We could possibly replace some uses of `futures::channel::mpsc` with crossbeam as well later on if there are performance benefits.